cleanlab/datalab/internal/issue_manager/underperf_group.py

```diff
@@ -242,13 +301,12 @@ def _get_cluster_statistics(
         self,
         n_clusters: int,
         cluster_labels: npt.NDArray[np.int_],
-        cluster_obj: ClusterMixin,
         knn_graph: csr_matrix,
     ) -> Dict[str, Dict[str, Any]]:
         cluster_stats = {
             "clustering": {
-                "algorithm": cluster_obj.__class__.__name__,
-                "params": {**cluster_obj.get_params()},  # Only works for sklearn.cluster objects
+                "algorithm": "HDBSCAN",
```

Comment:
Can we switch to `"DBSCAN"` as a default algorithm while we try to address the graph connectedness issue?
```suggestion
                "algorithm": "DBSCAN",
```